activestorage/lib/active_storage/service.rb

```diff
@@ -35,8 +35,8 @@ module ActiveStorage
   # can configure the service to use like this:
   #
   #   ActiveStorage::Blob.service = ActiveStorage::Service.configure(
-  #     :Disk,
-  #     root: Pathname("/foo/bar/storage")
+  #     :local,
+  #     { local: {service: Disk,  root: "/tmp/foo/storage" %>}
```

Comment:
Hi @akankshasaxena thanks for the PR. `Disk` here needs to be a string because at least in the console this version will raise an uninitialized constant error. There's also a missing closing brace in your change.

```suggestion
  #     { local: { service: "Disk",  root: Pathname("/foo/bar/storage") %> } }
```

Reply:
`%>` is also unmatched here.